src/components/views/auth/InteractiveAuthEntryComponents.tsx

```diff
@@ -1,7 +1,5 @@
 /*
-Copyright 2016 OpenMarket Ltd
-Copyright 2017 Vector Creations Ltd
-Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
+Copyright 2016-2021 The Matrix.org Foundation C.I.C.
```

Comment:
There was no copyright for 2018, so it can't be transferred.

Reply:
I would like to understand the logic of this in more detail. I've asked for clarification in the room, so let's discuss it more there.